styles.css

```diff
@@ -0,0 +1,93 @@
+form{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  margin: 4% auto;
+}
+
+.history{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.expenses{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  font-weight: 300px;
+}
+
+ul li{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  gap: 15px;
+  padding: 10px;
+  border-bottom: 1px solid #ddd;
```

Comment:
try to use CSS variables for your colors. learn more here:
https://blog.logrocket.com/how-to-use-css-variables-like-a-pro/